content/profiles/deborah-kurata.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Deborah Kurata"
+profileimage: "/images/profile-pictures/deborah-kurata-profile.jpg"
+coverimage: "/images/profile-pictures/deborah-kurata-cover.jpg"
+tags: ["Angular", "JavaScript", "TypeScript", "CSharp", ".NET", "Speaking - in person", "Speaking - virtual", "Panels", "MVP", "GDE"]
+twitter: "deborahkurata"
+github: "DeborahK"
+twitch: ""
```

Comment:
Hi Deborah! If you could delete lines 8, 9 and 11 altogether, that would be great.
Thanks!